src/pgrepr/src/oid.rs

```diff
@@ -11,6 +11,66 @@
 
 //! PostgreSQL OID constants.
 
+// PostgreSQL builtin type OIDs
+pub const TYPE_ANY_OID: u32 = 2276;
```

Comment:
We have an `Oid` data type defined in `mz_repr::adt::Oid`, should we use that for all these?